src/AlgebraicGeometry/Schemes/CoveredSchemes/Objects/Properties.jl

```diff
@@ -53,7 +53,8 @@ function _jacobian_criterion(X::CoveredScheme{<:Field})
 
   dec_info = decomposition_info(default_covering(X))
   for (V, fs) in dec_info
-    R = base_ring(OO(V))
+    R_quotient = OO(V)
+    typeof(R_quotient)<:MPolyQuoRing ? R = base_ring(R_quotient) : R = R_quotient
```

Comment:
```suggestion
    R_quotient isa MPolyQuoRing ? R = base_ring(R_quotient) : R = R_quotient
```
small nitpick: this is the preferred way in julia land if I remember correctly